### PR TITLE
fix missing components in lwesp_ll_template.c

### PR DESCRIPTION
**Pull request by niedong**
It seems that something is missing in the template, I added them on. It's the same as #75

### CODE REVIEW

**Comment by niedong** [comment]
(Only the branch is different)

**Comment by MaJerle** [comment]
This is not template specific. It is an example for STM32 examples.

File: lwesp/src/system/lwesp_ll_template.c [hunk 1]
```diff
@@ -89,7 +99,7 @@ lwesp_ll_init(lwesp_ll_t* ll) {
     }
 
     /* Step 3: Configure AT port to be able to send/receive data to/from ESP device */
-    configure_uart(->uart.baudrate);            /* Initialize UART for communication */
```
Comment (niedong) [inline lwesp/src/system/lwesp_ll_template.c:92]: Why written as "`->uart.baudrate`" instead of "`ll->uart.baudrate`".

Comment (niedong) [reply lwesp/src/system/lwesp_ll_template.c:92]: Plus there's missing function definition` configure_uart`. At least a function body should be provided.

**Comment by niedong** [comment]
> This is not template specific. It is an example for STM32 examples.

Please see my comment in the code

**Review by MaJerle — APPROVED** [review]

**Comment by MaJerle** [comment]
OK for me to merge. But we are merging to develop branch instead. Master branch is only for *releases*

**Comment by niedong** [comment]
Got it. Thanks.

**Pull request merged**